cmd/cli/root.go

```diff
@@ -0,0 +1,94 @@
+package cli
+
+import (
+	"fmt"
+	"os"
+
+	mcobra "github.com/muesli/mango-cobra"
+	"github.com/muesli/roff"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+
+	"github.com/bom-squad/go-cli/cmd/cli/options"
+	"github.com/bom-squad/go-cli/pkg/log"
+)
+
+var version = "dev"
```

Comment:
we should consider having a semantic format default version

Reply:
`0.0.0-dev` as the default, GoReleaser sets the version, so it doesn't matter.